blocks/share-bar-block/index.story.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
Why not use MDX for this story?

Reply:
oh huh, yeah will do. didn't realize how you had used props here https://github.com/WPMedia/fusion-news-theme-blocks/pull/485/files#diff-0007af772d045a78287e16f4cb7d2517L2

Reply:
hmm that link takes me to file changes?

Reply:
oh how you imported data from helper jsx

Reply:
in the previous mdx story

Reply:
https://github.com/WPMedia/fusion-news-theme-blocks/blob/stable/blocks/share-bar-block/helper.jsx https://github.com/WPMedia/fusion-news-theme-blocks/blob/stable/blocks/share-bar-block/index.story.mdx

Reply:
so the `helper.jsx` file was used for knobs that can be tweaked in the stories iirc

Reply:
cool, will work on that -- thanks

Reply:
weird this didn't work as I'd expect: 


```

import { ShareBar } from "./features/share-bar/default";
import data from "./helper";
import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";


## Props

<Props of={ShareBar} />

## Stories

** Custom ShareBar **

<Preview>
  <Story name="Custom ShareBar">
    <ShareBar {...data} />
  </Story>
</Preview>

```


```
import { boolean, text } from '@storybook/addon-knobs';

const data = () => {
  const headlineString = text('headlineString', 'Man Bites Dog');
  const websiteDomain = text('websiteDomain', 'https://www.thesun.com/');
  const websiteUrl = text('websiteUrl', '/2019/07/15/global-kitchen-sink-article/');
  const websiteName = text('websiteName', 'The Sun');

  const emailInput = boolean('email', true);
  const facebookInput = boolean('facebook', true);
  const pinterestInput = boolean('pinterest', true);
  const twitterInput = boolean('twitter', true);
  const linkedInInput = boolean('linkedIn', true);

  const customFields = {
    email: emailInput,
    facebook: facebookInput,
    pinterest: pinterestInput,
    twitter: twitterInput,
    linkedIn: linkedInInput,
  };

  return ({
    customFields,
    headlineString,
    websiteDomain,
    websiteUrl,
    websiteName,
  });
};

export default data;

Reply:
<img width="1181" alt="Screen Shot 2020-10-15 at 9 24 34 PM" src="https://user-images.githubusercontent.com/5950956/96205688-d5825c00-0f2c-11eb-8e45-e022f2439b96.png">

Reply:
looks like there's some cool mdx support in version 6 of storybook

Reply:
https://github.com/storybookjs/storybook/issues/10066#issuecomment-635502815